html-forms/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+
+  <form>
+    <div>Email</div>
+    <input type="text" name="email"/>
+  </form>
+
+  <form>
+    <div>Why is coding cool?</div>
+    <select name="coding">
+      <option value="select an option">Select an option</option>
+      <option value="because it's interesting">Because it's interesting</option>
+      <option value="because it's fun">Because it's fun</option>
+      <option value="because it's powerful">Because it's powerful</option>
+    </select>
+  </form>
+
+  <form>
+    <div>Favorite Language (choose one)</div>
+    <input type="radio" name="language" value="HTML"/> HTML
+    <input type="radio" name="language" value="CSS"/> CSS
+    <input type="radio" name="language" value="JavaScript"/> JavaScript
+  </form>
+
+  <form>
+    <div>Would you like to sign up for more information?</div>
+    <input type="checkbox" name="more info" value="sign me up"/>Sign me up!
+  </form>
+
+  <form>
+    <div>Comments</div>
+    <textarea name="comments" cols="20" rows="8"></textarea>
```

Comment:
This textarea needs an `id` to correlate to the `for` attribute on it's corresponding `label` element

Reply:
This still needs to be addressed